Buildings/Controls/OBC/CDL/Integers/Sources/TimeTable.mo

```diff
@@ -0,0 +1,207 @@
+within Buildings.Controls.OBC.CDL.Integers.Sources;
+block TimeTable "Table look-up with respect to time with constant segments"
+
+  parameter Real table[:,:]
+    "Table matrix with time as a first table column (in seconds, unless timeScale is not 1) and Integers in all other columns";
+  parameter CDL.Types.Extrapolation extrapolation=CDL.Types.Extrapolation.HoldLastPoint
+    "Extrapolation of data outside the definition range";
+  parameter Integer offset[:]=fill(0, nout) "Offsets of output signals";
+  parameter Modelica.SIunits.Time timeScale=1
+    "Time scale of first table column. Set to 3600 if time in table is in hours";
```

Comment:
`timeScale` must have no units. The first table column has seconds, so scale is just a non-dimensional factor.

Reply:
I updated for the continuous time table block as well.